gslb/ingestion/fullsync.go

```diff
@@ -295,26 +296,31 @@ func clusterSync(ctrlList []*GSLBMemberController, gsCache *avicache.AviCache) {
 			continue
 		}
 
-		if gslbutils.GetCustomFqdnMode() {
-			hostRules, err := c.hrClientSet.AkoV1beta1().HostRules("").List(context.TODO(), metav1.ListOptions{})
-			if err != nil {
-				gslbutils.Errf("cluster: %s, error in fetching hostrules, %s", c.name, err.Error())
-				return
-			}
-			for _, hr := range hostRules.Items {
-				if isHostRuleAcceptable(&hr) {
-					gFqdn := hr.Spec.VirtualHost.Gslb.Fqdn
-					lFqdn := hr.Spec.VirtualHost.Fqdn
+		hostRules, err := c.hrClientSet.AkoV1beta1().HostRules("").List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			gslbutils.Errf("cluster: %s, error in fetching hostrules, %s", c.name, err.Error())
+			return
```

Comment:
@arihantg should we continue to next cluster, in case if there is an issue with one cluster to fetch hostrule objects?

Reply:
No. If we continue we might delete existing GSLB services or pool members.